src/main/java/com/reckue/post/transfers/RatingRequest.java

```diff
@@ -16,12 +16,12 @@
 public class RatingRequest {
 
     @NotNull
-    @ApiModelProperty(notes = "Database generated rating ID")
+    @ApiModelProperty(notes = "Database generated rating Id")
     private String id;
```

Comment:
You need to delete String id in Request, because it doesn't generates in request.